.tmux.conf

```diff
@@ -91,7 +91,7 @@ bind -r C-l next-window     # select next window
 bind Tab last-window        # move to last active window
 
 # toggle mouse
-bind m run "cut -c3- ~/.tmux.conf | sh -s _toggle_mouse"
+bind m run "cut -c3- ~/.tmux.conf | sh -s _toggle_mouse tmux_conf_theme_mouse"
```

Comment:
Damn can you please remove this change.
I'm the one who introduced it. I wanted it to be
```
bind m run "cut -c3- ~/.tmux.conf | sh -s _toggle_mouse $tmux_conf_theme_mouse"
```
So that `_toggle_mouse()` keeps displaying a message when `tmux_conf_theme_mouse` is defined to the empty string. 
But I changed my mind. No message just fine.

Can you please update the branch? Again, I'm sorry

Reply:
Pushed! Let me know how that looks.

Reply:
Looks good to me. Will give a last look tomorrow. Heading to bed now. I'll merge in the morning 👋

Reply:
I'm still hesitating about the mouse emoji in comment. As you can see, I started to progressively removing these in the sample `.tmux.conf.local` because people enable them and then they open issues about status line rendering having bugs 😅

Reply:
It has been removed :)